pkg/workloads/spark_job/spark_util.py

```diff
@@ -223,8 +223,20 @@ def ingest(ctx, spark):
 
```

Comment:
Spark dataframe provides the schema for you, you can use that to create a map:

```
inferred_col_type_map = {(c.name: c.dataType) for c in df.schema}
```

The inferred_col_type_map that maps column name to spark type. You can use this to convert to cortex type. Instead of calling infer_type on the first row of every column.